CMakeLists.txt

```diff
@@ -876,6 +876,7 @@ if(SWIFT_BUILD_STDLIB OR SWIFT_BUILD_SDK_OVERLAY)
 
   message(STATUS "Building Swift runtime with:")
   message(STATUS "  Leak Detection Checker Entrypoints: ${SWIFT_RUNTIME_ENABLE_LEAK_CHECKER}")
+  message(STATUS "  Differentiable Programming Support: ${SWIFT_ENABLE_EXPERIMENTAL_DIFFERENTIABLE_PROGRAMMING}")
```

Comment:
Isn't it misleading to put this message under `Building Swift runtime with:`? How about moving the message somewhere else?

Differentiable programming doesn't currently involve Swift runtime changes. I think of `_Differentiation` as more like an auxiliary library, similar to SDK overlays.

Reply:
Ah, I see what you mean.  Sure, I'll outdent it.